draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1079,15 +1079,32 @@ Decoy digests increase the size of the SD-JWT. The number of decoy digests (or w
 
 ## Unlinkability
 
-Colluding Issuer/Verifier or Verifier/Verifier pairs could link issuance/presentation
-or two presentation sessions to the same user on the basis of unique values encoded in the SD-JWT
-(Issuer signature, salts, digests, etc.).
-
-To prevent these types of linkability, various methods, including but not limited to the following ones can be used:
-
-- Use advanced cryptographic schemes, outside the scope of this specification.
-- Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
-
+Unlinkability is a property that prevents an adversary from associating two or
+more sessions of the same user. Two types of unlinkability are often considered:
```

Comment:
You're right, 'sessions' is not a concept we used previously. How about this?
```suggestion
Unlinkability is a property that prevents an adversary from associating credential
presentations in certain ways. Two types of unlinkability are often considered:
```

Reply:
`from associating credential presentations of the same user to know about the behavior of that user beyond that user's consent`?
Probably some sentence is needed on why associating is a problem - ie adversary being able to know more about the user than it is supposed to, I think?

Reply:
imo the threat is about distinguishability, it has to do with the anonymity set of the data subject... You might find t-closenss / or similar concepts helpful. https://en.wikipedia.org/wiki/T-closeness

Issuer / Verifier: Issuers are distinguishable, Subjects are distinguishable (signature correlation), Holders are not distinguishable (batch issuance).

Verifier / Verifier: Issuers are distinguishable, Subjects are not distinguishable (linked secret), Holders are not distinguishable (zkp).

In all cases, we assume multiple verifiers process presentations and are compromised, and then we evaluate the damage done to data subjects.

Reply:
.... what I said above is not right... I think "Issuer / Verifier" unlinkability is a bad name... I think the security property we care about there is timing, metadata analysis.

Reply:
![Screen Shot 2023-10-30 at 1 53 04 PM](https://github.com/oauth-wg/oauth-selective-disclosure-jwt/assets/8295856/9f03157d-d735-4012-bafb-9338f9a96cd4)

Not sure if this helps, but the reason Issuer / Verifier Unlinkability is not achievable is the signature on the credential... batch issuance does not solve this, it just makes the attacker count unique signatures that are observed in issuers and verifiers.

Reply:
I think these labels are possibly the wrong words to be using especially because "verifier / verifier" unlinkability seems stronger than issuer / verifier.... but is actually not.

Reply:
Its possible that "single use unlinkability" and "multiple use unlinkability" might be better.

Reply:
@OR13  I'll make a bigger change to this PR hopefully addressing some of your comments, but certainly not everything. I propose that we try to get to an "acceptable" description first, get that merged, and then dig into the details in a separate issue.